### ISSUE TITLE
UV-webpack : Iframe embed URL is incorrect

### ISSUE THREAD

**Issue by WilsonAndrewWilson** [UniversalViewer/universalviewer#744]
```
 universalviewer@4.0.0-pre.12
```

**I'm submitting a:**

- [x] bug report
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**

When choosing to embed an item in UV, the iframe URL is not using the defined UV directory as the base URL, but rather just adding the uv.html onto the end of the current path.

**Expected behavior:**

URL should be as defined from the UV options.

**Steps to reproduce:**

[example](https://collections.st-andrews.ac.uk/item/italian-pocket-calendar-for-1919/625387/viewer#?#viewer&c=&m=&s=&cv=&xywh=-756%2C-150%2C4956%2C2936)
click Embed
hen check the iframe src URL
It's linking to https://collections.st-andrews.ac.uk/item/italian-pocket-calendar-for-1919/625387/uv.html rather than https://collections.st-andrews.ac.uk/uv/uv.html)

**Related code:**

```
// insert any relevant code here
```

**Other information:**
I've set up a RedirectMatch regex for the time being to get around this so the Iframe URL on the example above will now redirect to the correct path (thought worth mentioning in case someone tried it!)

**Comment by stephenwf** [comment]
Looks like the configuration in the webpack branch has been commented out:
https://github.com/UniversalViewer/universalviewer/blob/webpack/src/modules/uv-shared-module/BaseExtension.ts#L909-L946

[Introduced here](https://github.com/UniversalViewer/universalviewer/commit/e1ba115c32ba561ac3848d376449716291b3fff8)

@demiankatz @edsilv do you remember why this was removed? Is it safe to put back in now?

**Comment by edsilv** [comment]
I've published v4.0.0-pre.47 with a fix.
